WordPress/src/main/java/org/wordpress/android/modules/WordPressGlideModule.java

```diff
@@ -0,0 +1,36 @@
+package org.wordpress.android.modules;
+
+import android.content.Context;
+
+import com.android.volley.RequestQueue;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.GlideBuilder;
+import com.bumptech.glide.Registry;
+import com.bumptech.glide.annotation.GlideModule;
+import com.bumptech.glide.integration.volley.VolleyUrlLoader;
+import com.bumptech.glide.load.model.GlideUrl;
+import com.bumptech.glide.module.AppGlideModule;
+
+import org.wordpress.android.WordPress;
+
+import java.io.InputStream;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@GlideModule
+public class WordPressGlideModule extends AppGlideModule {
+    @Inject @Named("custom-ssl") RequestQueue mRequestQueue;
```

Comment:
Just to double check - Are we using  `custom-ssl` here to make sure we can use Glide in the future on self hosted sites? or is there any other reason?

Reply:
Yes that's the only reason - I had `regular` originally but I figured it was setting up an unnecessary trap for future self-hosted site support.